cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ReplicationThrottleHelper.java

```diff
@@ -173,15 +179,28 @@ private void setFollowerThrottledReplicas(String topic, Set<String> replicas) {
   }
 
   private void setThrottledReplicas(String topic, Set<String> replicas, String configKey) {
```

Comment:
Don't we need a similar special treatment when removing throttled replicas from a topic? If the existing throttle is `*`, it should be guaranteed that it is not set by CC; hence, shouldn't we leave it as it is (i.e. untouched)?

Reply:
As for `If the existing throttle is *, it should be guaranteed that it is not set by CC`, I think my change ensures that. Did I misunderstood your point or miss anything?

```
    if (oldThrottledReplicas != null && oldThrottledReplicas.trim().equals("*")) {
      // The existing setup throttles all replica. So, nothing needs to be changed
      return;
    }
```

Reply:
Does `#removeThrottledReplicasFromTopic` take the right action (i.e. leave the value as `*`) when the relevant config was already `*`? I suspect that it doesn't.

* We are also missing a unit test for this scenario, can we add one?

Reply:
That's a good point. I will modify `removeThrottledReplicasFromTopic ` to handle the `*` case and add unit test.